src/Cosmos/ProtobufSerialization.cpp

```diff
@@ -335,6 +336,12 @@ std::string buildAuthInfo(const Proto::SigningInput& input, TWCoinType coin) {
             signerInfo->mutable_public_key()->PackFrom(pubKey, ProtobufAnyNamespacePrefix);
             break;
         }
+        case TWCoinTypeNativeInjective: {
```

Comment:
Is it possible to reuse evmos's switch case?

Reply:
I tried using it, but RPC throws error.